jaxbind/jaxbind.py

```diff
@@ -43,6 +43,27 @@
 
 
 def _exec_abstract(*args, _func: FunctionType, **kwargs):
+    """JAXbind internal: bridging the user abstract function to JAX
```

Comment:
Here and everywhere else:
* I think `JAXbind interal:` is redundant as the name of the function already indicates that it is hidden.
* user -> user-defined ?